package/kernel/linux/modules/netdevices.mk

```diff
@@ -1026,7 +1026,7 @@ define KernelPackage/e1000e
   DEPENDS:=@PCIE_SUPPORT +kmod-ptp
   KCONFIG:=CONFIG_E1000E
   FILES:=$(LINUX_DIR)/drivers/net/ethernet/intel/e1000e/e1000e.ko
-  AUTOLOAD:=$(call AutoProbe,e1000e)
+  AUTOLOAD:=$(call AutoLoad,35,e1000e,1)
```

Comment:
AutoProbe works with pre-init flag as well, lets not use AutoLoad if really not required